### PR TITLE
Quickstart for the MedTech service

### PR DESCRIPTION
**Pull request by msjasteppe**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by msjasteppe** [comment]
#sign-off

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by msjasteppe** [comment]
merge

**Comment by msjasteppe** [comment]
#sign-off

**Comment by msjasteppe** [comment]
#merge

**Comment by msjasteppe** [comment]
#sign-off

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            "name": "[guid(format('{0}-FhirWriter', resourceGroup().id))]",
+            "dependsOn": [
+                "[variables('fhirServiceResourceId')]",
+                "[variables('iotConnectorResourceId')]"
+            ],
+            "properties": {
+                "roleDefinitionId": "[variables('fhirWriterRoleId')]",
+                "principalId": "[reference(variables('iotConnectorResourceId'), '2022-05-15', 'Full').identity.principalId]"
+            },
+            "scope": "[variables('fhirServiceResourceId')]"
```
Comment (bmoore-msft) [inline quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json:192]: See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        },
+        {
+            "type": "Microsoft.Authorization/roleAssignments",
+            "apiVersion": "2020-08-01-preview",
+            "name": "[guid(format('{0}-FhirWriter', resourceGroup().id))]",
+            "dependsOn": [
+                "[variables('fhirServiceResourceId')]",
+                "[variables('iotConnectorResourceId')]"
+            ],
+            "properties": {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json:188]: add the principalType property

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        "basename": {
+            "minLength": 3,
+            "maxLength": 16,
+            "type": "String",
+            "metadata": {
+                "description": "Base name that is used to name provisioned resources. Should be alphanumeric, at least 3 characters and less than 16 characters."
+            }
+        },
+        "location": {
+            "defaultValue": "[resourceGroup().location]",
```
[the remark this answers is not in the document]
Comment (msjasteppe) [reply quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json]: Feedback addressed in new PR commit.

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json (hunk 1 above)
Comment (msjasteppe) [reply quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json]: Feedback addressed in new PR commit.

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+            "name": "[guid(format('{0}-EventHubDataReceiver', resourceGroup().id))]",
+            "dependsOn": [
+                "[variables('eventHubResourceId')]",
+                "[variables('iotConnectorResourceId')]"
+            ],
+            "properties": {
+                "roleDefinitionId": "[variables('eventHubReceiverRoleId')]",
+                "principalId": "[reference(variables('iotConnectorResourceId'), '2022-05-15', 'Full').identity.principalId]"
+            },
+            "scope": "[variables('eventHubResourceId')]"
```
[the remark this answers is not in the document]
Comment (msjasteppe) [reply quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json]: Feedback addressed in new PR commit.

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+        },
+        {
+            "type": "Microsoft.Authorization/roleAssignments",
+            "apiVersion": "2020-08-01-preview",
+            "name": "[guid(format('{0}-EventHubDataReceiver', resourceGroup().id))]",
+            "dependsOn": [
+                "[variables('eventHubResourceId')]",
+                "[variables('iotConnectorResourceId')]"
+            ],
+            "properties": {
```
[the remark this answers is not in the document]
Comment (msjasteppe) [reply quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json:203]: Feedback addressed in new PR commit.

File: quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json (hunk 2 above)
Comment (msjasteppe) [reply quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json:188]: Feedback addressed in new PR commit.

**Comment by azure-quickstarts** [comment]
@msjasteppe - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by msjasteppe** [comment]
#sign-off

**Pull request merged**